lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -176,6 +176,10 @@ def _create_node_os_disk(
         self, environment: Environment, log: Logger, node: Node
     ) -> None:
         node_context = get_node_context(node)
+        # Give read permissions before accessing the file.
+        self.host_node.tools[Chmod].chmod(
+            node_context.os_disk_base_file_path, "a+r", sudo=True
```

Comment:
LISA can be run directly on a dev box. So, you can’t assume that passwordless sudo is available on the host node.

Reply:
Thanks for pointing this out. This is only needed when the host is remote. Will add a condition for it.

Reply:
@cwize1 Please review once again.